biocrnpyler/compartments.py

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) 2020, Build-A-Cell. All rights reserved.
+# See LICENSE file in the project root directory for details.
+
+import warnings
+
+class Compartment():
+
+    """ A formal Compartment object for a Species in a CRN.
+     A Compartment must have a name. They may also have a spatial dimension (such as 2 for two-dimensional,
+     or 3 for three-dimensional) and the volume in litres. 
+     Note: The "default" keyword is reserved for BioCRNpyler allotting a default compartment. 
+     Users must choose a different string. 
+    """
+
+    def __init__(self, name: str, volume = 1e-6, spatial_dimensions = 3, **keywords):
```

Comment:
Will we need units here too, eventually?

Reply:
Added units to compartments. Thanks for the suggestion!